packages/docs-site/src/components/presenters/data-table/data-table.scss

```
@@ -0,0 +1,222 @@
+  border: none;
+  font-size: font-size(s);
+  color: $text-color;
+}
+
+.data-table__caption {
+  caption-side: top;
+  padding: spacing(4) 0 spacing(6);
+  text-align: left;
+  color: $heading-color;
+  font-weight: 700;
+  font-size: font-size(l);
+}
+
+.data-table__head {
+  display: none;
+}
+
+.data-table__heading {
+  text-align: left;
+  white-space: nowrap;
+  text-transform: uppercase;
+  color: $heading-color;
+}
+
+.data-table__btn-sort {
+  border: none;
+  display: none;
+}
+
+.data-table__row {
+  display: block;
+  padding: spacing(8) 0;
+  border-bottom: 1px dashed $border-color;
+  background-color: $background-color;
+
+  &:first-of-type {
+    padding-top: 0;
+  }
+
+  &:last-of-type {
+    padding-bottom: 0;
+    border-bottom: none;
+  }
+}
+
+.data-table__heading,
+.data-table__cell {
+  padding: spacing(4);
+}
+
+.data-table__cell {
+  display: flex;
+  align-items: center;
+  padding: 0 spacing(4) 0 0;
+  border: 1px solid $border-color;
+  border-bottom: none;
+
+  &::before {
+    content: attr(data-column);
+    display: inline-block;
+    flex: 0 0 10rem;
+    padding: spacing(4);
+    margin-right: spacing(4);
+    border-right: 1px solid $border-color;
+    font-weight: 700;
+    color: $heading-color;
+    text-transform: uppercase;
+  }
+
+  &:first-of-type {
+    padding: 0 0 spacing(4) 0;
+    border: none;
+    font-size: font-size(m);
+
+    &::before {
+      display: none;
+    }
+  }
+
+
+  &:nth-child(1) span,
+  &:nth-child(3) span,
+  &:nth-child(4) span {
+    font-family: 'Roboto Mono', monospace;
+  }
+
+  &:last-of-type {
+    border-bottom: 1px solid $border-color;
+    font-weight: 700;
+  }
+}
+
+@include breakpoint("m") {
+
+  .data-table {
+    min-width: 20rem;
+    border-collapse: separate;
+    border-spacing: 0;
+    border-radius: 5px;
+    box-shadow: 0 2px 2px 0 rgba(000, 000, 000, 0.05);
+    
+    
+    // Table border logic
+    //
+    // Defining all table border & border-radius logic within 
+    // the .data-table class. We have to target the last table
+    // rows and cells, with all border logic being set on the 
+    // table cells themselves, rather than the rows/table.
+   
+    tr {
+      
+      th, td {
+        border-right: 1px solid $border-color;
+        border-bottom: 1px solid $border-color;
+      }
+
+      th {
+        border-top: 1px solid $border-color;
+        &:first-child {
+          border-top-left-radius: 4px;
+        }
+        &:last-child {
+          border-top-right-radius: 4px;
+        }
+      }
+
+      td {
+        &:first-child {
+          border-right: 1px solid $border-color;
+          border-bottom: 1px solid $border-color;
+        }
+      }
+
+      th:first-child, td:first-child {
+        border-left: 1px solid $border-color;
+      }
+
+      // Target the last table row and set the 
+      // border radius on the first and last element
+      &:last-child {
+        td:first-child {
+          border-bottom-left-radius: 6px;
```

Should this be 4px and use `$border-radius`?

---

Good spot. You're totally right
```suggestion
          border-bottom-left-radius: $border-radius;
```